entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+git clone https://github.com/matplotlib/matplotlib
+cd matplotlib/
+pip install -ve .
+cd doc/
+make html
+python -m http.server -d build/html/
```

Comment:
I think we can avoid all the `cd`s here?
```suggestion
pip install -ve ./matplotlib
make -C matplotlib/doc html
python -m http.server -d matplotlib/doc/build/html/
```

Reply:
Works for me. I think it was my original implementation :)